### ISSUE TITLE
missing napari-czifle2 in conda recipe of devbio-napari

### ISSUE THREAD

**Issue by manerotoni** [haesleinhuepf/devbio-napari#11]
This  
`conda create --name devbio-napari-env python=3.9 devbio-napari -c conda-forge
conda activate devbio-napari-env
`
failed to install `napari-czifile2`
A manual install with 

`conda install -c conda-forge napari-czifile2` helped.

**Comment by haesleinhuepf** [comment]
Hi Antonio @manerotoni,

the conda package should update over night with the dependency included. Thanks again for reaching out! Let me know if there are any further issues!

Best,
Robert

**Comment by manerotoni** [comment]
Hi Robert, 
I  think the mamba recipe also misses the napari-czifile2 (I thought it is the same as conda?)

**Comment by haesleinhuepf** [comment]
Hey Antonio @manerotoni ,

it's the same as conda, yes, and it lives here:
https://github.com/conda-forge/devbio-napari-feedstock/blob/main/recipe/meta.yaml#L55

If you still experience issues, would you mind sharing details? E.g. what did you install and how? What was the output of the program and how do you know installation doesn't work? The output of `conda list` in your environment would be interesting too.

Let me know!

Best,
Robert

**Comment by manerotoni** [comment]
The two install are definitely very different. I have a Windows10 miniconda install. Just performed the test today 26.07.2022, 9.25 AM

### conda create

`conda create --name devbio-napari-env-conda python=3.9 devbio-napari -c conda-forge`

Package list is `conda env export  > dev-napari-env-conda.log` **384 packages**

[devbio-napari-env-conda.log](https://github.com/haesleinhuepf/devbio-napari/files/9187466/devbio-napari-env-conda.log)

### mamba create
`mamba create --name devbio-napari-env-mamba python=3.9 devbio-napari -c conda-forge`

Package list is  `conda env export  > dev-napari-env-conda.log` **360 packages**
[devbio-napari-env-mamba.log](https://github.com/haesleinhuepf/devbio-napari/files/9187469/devbio-napari-env-mamba.log)


The package installed and also the versions installed are quite different! 
This is just a short list
![image](https://user-images.githubusercontent.com/3718770/180947662-5dd633f8-6c70-43b8-a885-dc6b089b031d.png)

Very confusing.

**Comment by manerotoni** [comment]
mamba (0.24.0) and conda (4.13.0) do not give me any error. I am able to install missing packages using 
`mamba install ... -c conda-forge`


In the report of what is being installed packages are indeed missing:

Here part of the mamba install log

![image](https://user-images.githubusercontent.com/3718770/180949608-f378538e-cfef-4955-be94-394e6e4df357.png)

and here the corresponding part with conda

![image](https://user-images.githubusercontent.com/3718770/180949723-39293620-54a1-4c68-820c-b23b4fdf7ea1.png)

**Comment by haesleinhuepf** [comment]
> The package installed and also the versions installed are quite different!

Indeed! Conda installs devbio-napari 0.5.9 and mamba installs devbio-napari 0.5.6 .

What happens if you explicitly install a specific devbio-napari version? Do conda and mamba do the same then?

Furthermore, did you install mamba in the base environment? And did you call both installations from the base?

**Comment by manerotoni** [comment]
conda and mamba are in the base environment. 
But why do they use a different distribution? 
I would think that they should both use the latest.
I will try to install 0.5.9, then I expect to get the same result.

**Comment by manerotoni** [comment]
From base environment (mamba installed with `conda install mamba`)

`conda create --name devbio-napari-env-conda-0.5.9 python=3.9 devbio-napari=0.5.9 -c conda-forge`
[devbio-napari-env-conda-0.5.9.log](https://github.com/haesleinhuepf/devbio-napari/files/9196523/devbio-napari-env-conda-0.5.9.log)
384 packages installed

`mamba create --name devbio-napari-env-mamba-0.5.9 python=3.9 devbio-napari=0.5.9 -c conda-forge`
[devbio-napari-env-mamba-0.5.9.log](https://github.com/haesleinhuepf/devbio-napari/files/9196527/devbio-napari-env-mamba-0.5.9.log)
390 packages installed

We still have a delta of 6 packages, but now in the inverse direction, also some packages have a different versions.

I think these packages are unrelated to devbio-napari.

**Comment by haesleinhuepf** [comment]
Hi @manerotoni ,

I suggest to talk to the conda and/or mamba developers about this. Furthermore, we are planning to release devbio-napari in the future with pinned dependencies. It should then become more robust against variations.

Best,
Robert

**Comment by manerotoni** [comment]
Will you communicate with conda/mamba developers or should I do it?
It is a little scary that 2 tools that are meant to do the same install do something quite different. Mamba  resolve dependencies differently and this explains why you are faster but may also get you two different results.
